packages/extension/src/inpage/index.ts

```diff
@@ -70,7 +70,7 @@ window.addEventListener(
     }
 
     if (data.type === "DISCONNECT_ACCOUNT") {
-      await alephium.disconnect()
+      await (alephium as any)._disconnect()
```

Comment:
We don't need to remove the preauthorization here, because it has already been removed.